frontend/server/src/DAO/Runs.php

```diff
@@ -934,7 +934,8 @@ final public static function getForProblemDetails(
      * Returns the time of the next submission to the current problem
      */
     final public static function nextSubmissionTimestamp(
-        ?\OmegaUp\DAO\VO\Contests $contest
+        \OmegaUp\DAO\VO\Contests $contest = null,
+        \OmegaUp\Timestamp $lastSubmissionTime = null
```

Comment:
```suggestion
        ?\OmegaUp\DAO\VO\Contests $contest = null,
        ?\OmegaUp\Timestamp $lastSubmissionTime = null
```

aquí los tipos dicen que no puede ser nulo, pero se les está asignando `null`, lo cual es contradictorio. (no se por qué Psalm acepta que esto suceda :S tendré que investigar)